compiler+runtime/src/cpp/jank/read/lex.cpp

```diff
@@ -266,7 +287,8 @@ namespace jank::read
       }
 
       auto const token_start(pos);
-      switch(file[token_start])
+      auto a = file[token_start];
```

Comment:
1. Please use direct initialization. i.e. `auto a(...);`
2. Please make this const, if it's not being mutated
3. I'm not sure what `a` stands for; if it's just a lazy name, please it something more clear like `current_char`
4. I don't actually see this local being used anywhere else; is it needed?

Reply:
sorry this is for debugging purpose. forgot to remove it. it's removed now.